README.md

```diff
@@ -1,3 +1,5 @@
+[![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/4906/badge)](https://bestpractices.coreinfrastructure.org/projects/4906)
```

Comment:
Hmmm this is failing mdlint - maybe place this
```
<!-- markdownlint-disable -->
```
just above this line to ignore it?

Reply:
And on that note - do we want badges at the top? Or would it be better just below the gif on line 18?

Reply:
Or it could work to move the badge as the first thing right after the page title header.

If we want to keep it here, place `<!-- markdownlint-disable MD041 -->` on the very first line so we only disable this check.

Reply:
I tried having that line first and the image didn't appear. I'll try moving it.

Reply:
It worked that time.